recipes/pkg-config/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: pkg-config
+  version: 0.28
+
+source:
+  fn: pkg-config-0.28.zip
+  url: https://github.com/tpn/pkg-config/archive/pkg-config-0.28.zip
+
+build:
+  skip: True  #[win]
+  number: 2
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - zlib
+    - autoconf
+    - python
+    - automake
+    - libtool
+  run:
+    - zlib
+
+test:
+  commands:
+    - pkg-config --help
+    - !!str [ "$(pkg-config freetype2 --cflags)" == "-I$PREFIX/include/freetype2 -I$PREFIX/include" ]
+
+  requires:
+    - freetype
+
+about:
+  home: http://www.freedesktop.org/wiki/Software/pkg-config/
+  license: GPL 2
```

Comment:
I suspect you will get lint about not having a summary (https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/lint_recipe.py#L25)